src/android/CodePush.java

```diff
@@ -616,6 +631,37 @@ public void onStart() {
         }
     }
 
+    private Boolean hasIonicWebViewEngine() {
+        try {
+            Class.forName("com.ionicframework.cordova.webview.IonicWebViewEngine");
+            return true;
+        } catch (ClassNotFoundException e) {}
+        return false;
+    }
+
+    private void setServerBasePath(String serverPath) {
+        try {
+          Class ionicWebViewEngineClass = Class.forName("com.ionicframework.cordova.webview.IonicWebViewEngine");
+          Object ionicWebViewEngine = ionicWebViewEngineClass.cast(this.mainWebView.getEngine());
+          Method setServerBasePath = ionicWebViewEngineClass.getMethod("setServerBasePath", String.class);
+
+          final String ionicWebViewEngineServerPath = serverPath;
+          this.cordova.getActivity().runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+              try {
+                setServerBasePath.invoke(ionicWebViewEngine, ionicWebViewEngineServerPath);
+              } catch (IllegalAccessException | InvocationTargetException e) {
+                e.printStackTrace();
+              }
+            }
+          });
+          return;
```

Comment:
The same as in https://github.com/Microsoft/cordova-plugin-code-push/pull/454/files#diff-a02a5065115d3f2b6927f7938b46e092R659